upup/pkg/fi/cloudup/terraform/literal.go

```diff
@@ -48,6 +49,14 @@ type Literal struct {
 	FilePath string `cty:"file_path"`
 	// FileFn represents the function used to reference the file
 	FileFn fileFn `cty:"file_fn"`
+	// CidrSubnetFn represents the function used calculate the subnetMask for ipv6 aws vpc
+	CidrSubnetFn string `cty:"cidrsubnet_fn"`
```

Comment:
```suggestion
	CIDRSubnetFn string `cty:"cidrsubnet_fn"`
```
but why is this a field?